pkg/reconciler/cluster/syncer/syncer_controller_start.go

```diff
@@ -37,6 +38,7 @@ func BindOptions(o *Options, fs *pflag.FlagSet) *Options {
 	fs.StringVar(&o.SyncerImage, "syncer-image", o.SyncerImage, "Syncer image to install on clusters")
 	fs.BoolVar(&o.PullMode, "pull-mode", o.PullMode, "Deploy the syncer in registered physical clusters in POD, and have it sync resources from KCP")
 	fs.BoolVar(&o.PushMode, "push-mode", o.PushMode, "If true, run syncer for each cluster from inside cluster controller")
+	fs.StringVar(&o.KCPHost, "kcp-host", o.KCPHost, "The KCP Api-server host in host:port format.")
```

Comment:
API server

Wondering, "host" suggests that this is a hostname. But it isn't. It's an https URL, no? Maybe `--kcp-address` or `--kcp-url`? @marun opinions?